CITATION.cff

```diff
@@ -1,40 +1,75 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
 cff-version: 1.2.0
-message: "If you use this software, please cite it as below."
-url: "https://github.com/helmholtz-analytics/heat"
+title: HeAT
+message: 'If you use this software, please cite it as below.'
+type: software
+authors:
+  - family-names: Götz
+    given-names: Markus
+  - family-names: Debus
+    given-names: Charlotte
+  - family-names: Coquelin
+    given-names: Daniel
+  - family-names: Krajsek
+    given-names: Kai
+  - family-names: Comito
+    given-names: Claudia
+  - family-names: Knechtges
+    given-names: Philipp
+  - family-names: Hagemeier
+    given-names: Björn
+  - family-names: Tarnawa
+    given-names: Michael
+  - family-names: Hanselmann
+    given-names: Simon
+  - family-names: Siggel
+    given-names: Martin
+  - family-names: Basermann
+    given-names: Achim
+  - family-names: Streit
+    given-names: Achim
```

Comment:
```suggestion
title: Heat
message: 'If you use this software, please cite it as below.'
type: software
authors:
# core team
  - family-names: Comito
    given-names: Claudia
  - family-names: Coquelin
    given-names: Daniel
  - family-names: Debus
    given-names: Charlotte
  - family-names: Götz
    given-names: Markus
  - family-names: Gutiérrez Hermosillo Muriedas
     given-names: Juan Pedro
  - family-names: Hagemeier
     given-names: Björn
  - family-names: Hoppe
     given-names: Fabian
  - family-names: Knechtges
    given-names: Philipp
  - family-names: Krajsek
    given-names: Kai
  - family-names: Rüttgers
     given-names: Alexander    
  - family-names: Tarnawa
    given-names: Michael
# release contributors - add as needed    
```

Reply:
@JuanPedroGHM I see the authors field is mandatory now. Here's my suggestion regarding the list of authors, reflecting 2023 status instead of 2018 :) I hope I'm not forgetting anybody. 

I also think we should add open-source contributors to the list of authors before every release, as long as it is sustainable.